### ISSUE TITLE
Invalid redirect URL when signing-out of demo application

### ISSUE THREAD

**Issue by esune** [bcgov/vc-authn-oidc#310]
When walking through the demo, at the time of signing-out of the demo Keycloak displays an error message stating "Invalid redirect URL" and fails to complete the logout process. This may be related to a recent change that upgraded Keycloak from v16 to v22, so we may be missing a configuration setting somewhere - see https://www.keycloak.org/guides#server for documentation on how to configure Keycloak.

**Comment by swcurran** [comment]
I found this in googling -- https://github.com/dsb-norge/vue-keycloak-js/issues/141#issuecomment-1129568131 that suggest that adding this option `spi-login-protocol-openid-connect-legacy-logout-redirect-uri=true` might address the issue.  Added the equivalent environment variable to the docker-compose file:

```
    KC_SPI_LOGIN_PROTOCOL_OPENID_CONNECT_LEGACY_LOGOUT_REDIRECT_URI: true
```

And the flow worked.  Seems weird, but I think that solves it.  I did find a few other things to try, but they didn't seem to impact it.

**Comment by swcurran** [comment]
BTW -- once the redirection is done, the title of the page is "undefined", which seems bad.

**Comment by popkinj** [comment]
> BTW -- once the redirection is done, the title of the page is "undefined", which seems bad.

Wouldn't that be an issue with the page you're getting redirected too?

**Comment by swcurran** [comment]
> Wouldn't that be an issue with the page you're getting redirected too?

Yes — but that app is part of the vc-auth-oidc.  I’ll open another PR. Same thing when first loading the vue app.
